platforms/android/example-compose/src/main/java/io/element/wysiwyg/compose/MainActivity.kt

```diff
@@ -33,6 +34,7 @@ class MainActivity : ComponentActivity() {
                         RichTextEditor(
                             state = state,
                             modifier = Modifier.fillMaxWidth(),
+                            style = RichTextEditorDefaults.style(),
                         )
```

Comment:
Can't we use the default value instead?

Reply:
That's true although, this being an example app, I wanted to show this being set explicitly. Perhaps it'd make more sense to use a non-default value here though!